builtin/providers/google/resource_dns_managed_zone.go

```diff
@@ -32,7 +34,6 @@ func resourceDnsManagedZone() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 				ForceNew: true,
-				Default:  "Managed by Terraform",
```

Comment:
So I think if you leave the Default as it is and set this to `Computed: true`, things _should_ be OK and not cause problems. What do you think?